webapp/components/FollowButton.vue

```diff
@@ -75,7 +75,7 @@ export default {
 
         const followedUser = follow ? data.followUser : data.unfollowUser
         this.$emit('update', followedUser)
-      } catch {
+      } catch (err) {
```

Comment:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-26T06:47:21Z" title="Thursday, September 26th 2019, 8:47:21 am +02:00">Sep 26, 2019</time>_

---
> **Outdated (history rewrite)** - original diff
---
```diff
@@ -75,7 +75,7 @@ export default {
 
         const followedUser = follow ? data.followUser : data.unfollowUser
         this.$emit('update', followedUser)
-      } catch {
+      } catch (err) {
```


code creep :cold_sweat: 
I guess adding this doesn't hurt at all, even though we don't use `err`
and we are all certainly guilty of adding small changes we notice that have nothing to do with the ticket we are working on, but it's something we are trying to build a culture of not doing.

Reply:
> [<img alt="vbelolapotkov" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/vbelolapotkov) **Authored by [vbelolapotkov](https://github.com/vbelolapotkov)**
_<time datetime="2019-09-26T07:24:58Z" title="Thursday, September 26th 2019, 9:24:58 am +02:00">Sep 26, 2019</time>_

---


@mattwr18 I agree this is not the part of the ticket. I understand and support the intention of building such a culture. So I need your advice here on how to properly solve this.

Here is the problem I've encountered.
The PR failed and I wasn't able to check in github (at least I haven't found any detailed report) which exactly tests failed. So I ran all the tests locally and fixed the failed ones :)

IMO this one micro change should be merged into master before the rest of PR.

P.s. I'm surprised how other PRs are not failing.

Reply:
> [<img alt="mattwr18" height="40" width="40" align="left" src="https://avatars1.githubusercontent.com/u/38261864?s=88&v=4">](https://github.com/mattwr18) **Authored by [mattwr18](https://github.com/mattwr18)**
_<time datetime="2019-09-27T07:09:00Z" title="Friday, September 27th 2019, 9:09:00 am +02:00">Sep 27, 2019</time>_

---


Are you not able to access the builds in Travis? I either click on the link in the `PR` itself, or go to https://travis-ci.com/Human-Connection/Human-Connection/builds and sign in with my GitHub account and then I can see all the builds and investigate why it failed...
I think we can just leave the change here, I'm happy you understand and support the intention, I'm certainly guilty of code creep from time to time so feel free to call me out on it. I think just raising the conversation will help us in the long run.